src/list/manage/containers/modals.css

```diff
@@ -0,0 +1,120 @@
+.connectDevice {
```

Comment:
nit: the convention with class names is "hyphen-case"; the CSS bundling plugin for WebPack turns this into `camelCase` in the code.

Reply:
Good catch, thanks! I'll fix